content/agent/faq/datadog-ip-ranges.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Datadog IP ranges
+kind: faq
+---
+
+All data sent to Datadog or coming from Datadog Webhooks and integrations, such as those with ticketing systems like ServiceNow and JIRA, are coming from a set of static IPs:  
```

Comment:
Should `Webhooks` be capitalized?

Reply:
Should it be this instead?

> All data sent to Datadog or coming from Datadog webhooks and integrations, such as those with ticketing systems like ServiceNow and JIRA, are coming from a set of static IPs:

Reply:
This statement is confusing and wrong (as it's currently worded). `All data sent to Datadog... are coming from a set of static IPs`. That makes it sound like the Datadog Agent is sending *from* those IPs, when it's actually sending *to* those IPs.

Do we really need this placeholder page or can we just link the Agent FAQ directly to the wiki page (https://github.com/DataDog/dd-agent/wiki/Network-Traffic-and-Proxy-Configuration)? If we link directly we don't have to duplicate content and it saves the user one click to get to the pertinent info.